develop/dev-guide-bookshop-schema-design.md

```diff
@@ -78,34 +98,40 @@ On the database details page of TiDB Cloud, you can click the **Import** button
 
    **Role-ARN**:
 
-   {{< copyable "" >}}
+    {{< copyable "" >}}
+
+    ```
+    arn:aws:iam::494090988690:role/s3-tidb-cloud-developer-access
+    ```
 
-   ```
-   arn:aws:iam::494090988690:role/s3-tidb-cloud-developer-access
-   ```
+    In this example, the following data is generated in advance:
 
-   In this example data, 200,000 user information, 500,000 book information, 100,000 author information, 1 million rating records, and 1 million order information are pre-generated.
+    - 200,000 rows of user information
+    - 500,000 rows of book information
+    - 100,000 rows of author information
+    - 1,000,000 rows of rating records
+    - 1,000,000 rows of order records
 
-2. Select **Bucket Region** as **US West (Oregon)**.
-3. Select **Data Format** as **TiDB Dumpling**.
+2. Select **US West (Oregon)** for **Bucket Region**.
+3. Select **TiDB Dumpling** for **Data Format**.
 
-   ![Import Bookshop data in TiDB Cloud](/media/develop/tidb_cloud_import_bookshop_data.png)
+    ![Import Bookshop data in TiDB Cloud](/media/develop/tidb_cloud_import_bookshop_data.png)
 
-4. Enter your database login information.
-5. Click the **Import** button to confirm the import.
-6. Wait for TiDB Cloud to complete the data import.
+4. Enter database login information.
+5. Click the **Import** option to confirm the import.
```

Comment:
I think it's a button, not an option.

Reply:
> I think it's a button, not an option.
Please see the word list of [Google developer documentation style guide](https://developers.google.cn/style/word-list#letter-b)
![image](https://user-images.githubusercontent.com/89768198/169734375-711f3f75-ae9e-40da-9f1a-00f6aad01c4c.png)

Reply:
Let us look at this guide in conjunction with practical examples:

https://cloud.google.com/s/results?q=option
https://cloud.google.com/s/results?q=button

I think this document ignores the button description of the PC side, and I believe **Import** is an on-screen button.